src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,71 @@
+const expensesServices = require('../services/expenses/expenses.services');
+const { isExpenseValid } = require('../untils/expenses');
+const { isUserExist } = require('../untils/users');
+
+const getExpenses = (req, res) => {
+  const { categories, userId, from, to } = req.query;
+
+  return res
+    .status(200)
+    .json(expensesServices.getExpenses(categories, userId, from, to));
+};
+
+const getExpenseById = (req, res) => {
+  const { id } = req.params;
+
+  const expenseId = Number(id);
+
+  if (!expenseId || isNaN(expenseId)) {
```

Comment:
The check `!expenseId || isNaN(expenseId)` is redundant because `Number(id)` will always return a number, and `isNaN(expenseId)` will suffice to check if the conversion was successful. Consider simplifying this condition to just `isNaN(expenseId)`.